variables.tf

```diff
@@ -8,6 +8,11 @@ variable "ecs_cluster_arn" {
   description = "The ARN of the ECS cluster where service will be provisioned"
 }
 
+variable "ecs_cluster_name" {
```

Comment:
I would prefer you use a data resource with the provided ARN to get the name. Otherwise we can have a situation where the arn and name don't match.

Reply:
Yea I was kinda wondering if there was a better way to grab that info, and I think you're right I'd rather go with a data resource too.